spec/acceptance/log_from_organizer_spec.rb

```diff
@@ -59,9 +59,9 @@ def collects_log
       expect(log_message).to include(organizer_log_message)
     end
 
-    it "lists the keys in contect after the actions are executed" do
+    it "lists the keys in context after the actions are executed" do
       organizer_log_message = "[LightService] -     keys in context: " \
-                              ":tea, :milk, :coffee, :milk_tea, :latte"
+                              ":tea, :milk, :coffee, :_around_actions, :milk_tea, :latte"
```

Comment:
This is internal to LS, we should not have this visible in the `Context`. What do you think about adding this field to the `Context`? Like we do with `current_action`, [here](https://github.com/adomokos/light-service/blob/0a70d929e6f0b1465ba3cfaede74511f39a125db/lib/light-service/context.rb#L11).

Reply:
Agree, `_around_actions` is internal. 
I'm not sure I understand your phrase and a little confused: if we explicitly add it to `attr_accessor` it's a public attribute. But you also saying that we need to hide it. Do you have an example in mind how to achieve invisibility?